com.unity.render-pipelines.high-definition/Documentation~/Ray-Traced-Reflections.md

```diff
@@ -86,3 +86,6 @@ When using quality mode, there are extra properties that you can use to customiz
 | ---------------- | ------------------------------------------------------------ |
 | **Sample Count** | Controls the number of rays per pixel per frame. Increasing this value increases execution time linearly. |
 | **Bounce Count** | Controls the number of bounces that reflection rays can do. Increasing this value increases execution time exponentially. |
+
+## Limitations
+Currently, ray-traced reflections do not support [decals](decal.md). This means that, when you use ray-traced reflection, decals do not appear in reflective surfaces.
```

Comment:
It is not only raytrace reflections, it is all the effects: This include RTGI.
So it is mainly RTGI and RTR. Guess we should add the information in both pages of this effects + on the limitation page of the getting started with raytracing? thanks

Reply:
I've added the info to the getting started page and rtgi page, how is it now?